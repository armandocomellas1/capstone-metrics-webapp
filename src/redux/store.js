import { configureStore } from '@reduxjs/toolkit';
import globalSlice from './global/globalReduce';

const reducer = {
  globaStats: globalSlice,
};

const store = configureStore({
  reducer,
  devTools: process.env.NODE_ENV !== 'production',
});

export default store;
