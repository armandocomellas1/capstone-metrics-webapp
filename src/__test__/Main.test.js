import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MainStat from '../components/Main';
import store from '../redux/store';

describe('Main page component testing using snapshots', () => {
  it('Test Main component renders', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MainStat />
        </Provider>,
      );
    expect(tree).toMatchSnapshot();
  });
});
