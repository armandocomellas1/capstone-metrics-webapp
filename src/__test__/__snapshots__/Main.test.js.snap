// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page component testing using snapshots Test Main component renders 1`] = `null`;
